Javascript(ES6)/README.md

```diff
@@ -3126,9 +3126,8 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
 **Tools**
 
   - Code Style Linters
-    + [ESlint](http://eslint.org/) - [Airbnb Style .eslintrc](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc)
+    + [ESlint](http://eslint.org/) - [Making Sense Style .eslintrc](.eslintrc)
     + [JSHint](http://jshint.com/) - [Airbnb Style .jshintrc](https://github.com/airbnb/javascript/blob/master/linters/.jshintrc)
```

Comment:
I don't think JSHint has been deprecated yet. I was planning on adding a new file for it -- should we use it at all? I think for most cases eslint should already cover it.

What does everyone think?